docs/setup-cluster/checklists/_index.rst

```diff
@@ -315,11 +315,47 @@ Non-Root Containers
 ===================
 
 You can enhance security and limit potential malicious activity by running containers as non-root
-users. To find out more, visit :ref:`run-as-user` and :ref:`run-unprivileged-tasks`.
+users.
 
 .. important::
 
-   Red Hat OpenShift users should not configure non-root containers.
+   Red Hat® OpenShift® users users should not configure non-root containers.
```

Comment:
this deserves an explanation as to why: "... should not configure non-root containers according to these instructions because OpenShift configures containers to run as non-root in a manner which conflicts with Determined's non-root strategy".

Or something like that.